src/dex-ui/pages/dao/DAODetailsPage/MultiSigDAODashboard/Settings.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button, Divider, Flex, SimpleGrid, Tag, Text, IconButton, Image } from "@chakra-ui/react";
+import { CopyIcon, DeleteIcon } from "@chakra-ui/icons";
+import { useForm } from "react-hook-form";
+import { FormInput } from "@dex-ui-components";
+import { RefreshIcon } from "@dex-ui-components";
+import { Color } from "@dex-ui-components";
+import * as R from "ramda";
+import { Member } from "./MultiSigDAODashboard";
+import { MultiSigDAODetails } from "@services";
+import { DAOFormContainer } from "../../CreateADAO/forms/DAOFormContainer";
+import { isValidUrl } from "@dex-ui/pages/CreateProposal/utils";
```

Comment:
You should be able to import from `@pages` if the utils file is exported in index.ts

Reply:
Done